pkg/ipfs/node.go

```diff
@@ -93,6 +84,7 @@ func NewNodeWithConfig(ctx context.Context, cm *system.CleanupManager, cfg types
 		}
 	}()
 
+	// TODO if cfg.PrivateInternal is true do we actually want to connect to peers?
 	if err = connectToPeers(ctx, api, ipfsNode, cfg.GetSwarmAddresses()); err != nil {
 		log.Ctx(ctx).Error().Msgf("ipfs node failed to connect to peers: %s", err)
 	}
```

Comment:
I left a comment in https://expanso.atlassian.net/browse/ENG-109?focusedCommentId=10143 about this. Feels like we should **not** be establishing connections with other nodes if we want a "private" ipfs node. However, that would make downloading results from compute nodes which are not running in the same process basically impossible. Perhaps we could rename the two ipfs related flags:
- `ipfs-internal`: spins up an in-process node for bacalhau.
- `ipfs-remote/external`: tells bacalhau to use an remote/external ipfs node.